src/dockerflow/sanic/checks.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+This module contains built-in checks for the Sanic integration.
+"""
+from .. import health, messages
+
+
+async def check_aioredis_connected(app):
+    """
+    A built-in check to connect to Redis using the given client and see
+    if it responds to the ``PING`` command.
+
+    It's automatically added to the list of Dockerflow checks if a
+    :class:`~aioredis.Redis` instance is passed
+    to the :class:`~dockerflow.sanic.app.Dockerflow` class during
+    instantiation, e.g.::
+
+        import aioredis
+        from sanic import Sanic
+        from dockerflow.sanic import Dockerflow
+
+        app = Sanic(__name__)
+        app.config['REDIS'] = {'address': 'redis://:password@localhost:6379/0'}
+
+        @app.listener('before_server_start')
+        async def aio_redis_configure(app, loop):
+            app.redis = aioredis.create_redis(loop=loop, **app.config['REDIS'])
+
+        @app.listener('after_server_stop')
+        async def close_redis(_app, _loop):
+            _app.redis.close()
+            await _app.redis.wait_closed()
+
+        dockerflow = Dockerflow(app, aioredis=True)
+
+    An alternative approach to instantiating a Redis client directly
+    would be using the `Sanic-Redis <https://github.com/strahe/sanic-redis>`_
+    Sanic extension::
+
+        from sanic import Sanic
+        from sanic_redis import SanicRedis
+        from dockerflow.sanic import Dockerflow
+
+        app = Sanic(__name__)
+        app.config['REDIS'] = {'address': 'redis://:password@localhost:6379/0'}
+        SanicRedis(app)
+
+        dockerflow = Dockerflow(app, aioredis=True)
+
+    """
+    import aioredis
+    errors = []
+
+    try:
+        result = await app.redis.ping()
```

Comment:
Expecting `app.redis` is a bad assumption and we should follow the idea of simply coupling the check to one specific Sanic extension, such as [sanic-redis](https://github.com/strahe/sanic-redis). See above for more details.